Makefile

```diff
@@ -0,0 +1,15 @@
+# Currently this just contains the existing docs build cmds from docker/docker
```

Comment:
If possible, I'd like to avoid a root `Makefile` (and having a `docs/Makefile` instead).

This is because it goes against `golang` conventions: a proper go project should be buildable with `go build` and testable with `go test` out of the box, no Makefile required.

As a reference, none of the official `golang` repositories have a `Makefile`:
https://github.com/golang